lib/generators/draft/resource/templates/specs/factories.rb

```diff
@@ -1,11 +1,10 @@
 # Copy this file into your spec/support folder; create one if you don't have it.
 # https://github.com/firstdraft/increasing_random/blob/master/increasing_random.rb
 
-require Rails.root.join("spec", "support", "increasing_random.rb")
 
-FactoryGirl.define do
+FactoryBot.define do
   factory :<%= singular_table_name %> do
-    sequence(:id, IncreasingRandom.new) { |n| n.value }
+    sequence(:id) { |n| n }
```

Comment:
I just realized that this is probably no longer necessary, since the database will do increasing integer IDs automatically.

Reply:
👍